charts/catalog/values.yaml

```diff
@@ -54,6 +54,18 @@ apiserver:
       useEmbedded: true
       # etcd URL(s); override this if NOT using embedded etcd
       servers: http://localhost:2379
+      # etcd persistence options IF using embedded etcd
+      persistence:
+        enabled: true
```

Comment:
Let's make this disabled by default so that we don't drop a big change onto people that don't expect it. We should advertise that the chart now supports this, so that people without a lot of familiarity with helm can have some runway to get up to speed before it becomes the default (if ever).

I'm LGTM on this change if the default is disabled (no need to change accessMode or size, so you can just turn it on and it works).